docs/help/specifications-support/index.md

```diff
@@ -0,0 +1,14 @@
+# Specifications Support
+
+Bump.sh supports the most popular specifications for REST APIs and Event-Driven APIs.
+
+```mdx-code-block
+import DocCardList from '@theme/DocCardList';
+
+<DocCardList />
+```
```

Comment:
Why do we keep this? It seems redundant with the lines coming just after. Anything I'm missing here?

<img width="856" alt="Capture d’écran 2023-02-09 à 09 31 02" src="https://user-images.githubusercontent.com/30788/217758329-e01fa4c1-9f2a-4506-a409-e0bededace39.png">

Reply:
The goal of the cards was to remove the hand written list links underneath.

Reply:
WDYT of keeping the handwritten links for now, to keep things consistent with the other pages, and then think about how we could use these cards more globally? @jayk-u?

Reply:
Ok no problemo. I'll approve the PR once the `DocCardList` component is removed then :)